scripts/demo.sh

```
@@ -96,50 +96,582 @@ else
+        echo -e "${CYAN}━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━${NC}"
+        echo ""
+        echo -e "  Commands: ${GREEN}[amount]${NC} deposit | ${YELLOW}[-amount]${NC} withdraw (future)"
+        echo -e "            ${BLUE}[T]${NC}enant | ${BLUE}[P]${NC}arty | ${BLUE}[A]${NC}ccount | ${BLUE}[Q]${NC}uit"
+        echo ""
+        echo -e -n "${CYAN}Enter command: ${NC}"
+        read -r CMD
+
+        case "$CMD" in
+            [Tt])
+                echo ""
+                return "tenant"
```

Shellcheck (0.11.0)

[error] Can only return 0-255. Other data should be written to stdout.

(SC2152)